CHANGELOG.next.asciidoc

```diff
@@ -4,94 +4,180 @@
 :pull: https://github.com/elastic/beats/pull/
 
 === Beats version HEAD
-https://github.com/elastic/beats/compare/v6.6.2...6.6[Check the HEAD diff]
+https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD diff]
 
 ==== Breaking changes
 
 *Affecting all Beats*
 
+- Update to Golang 1.12.1. {pull}11330[11330]
```

Comment:
You are not pulling in all these changes, are you?
This file may be auto-generated. I'm not sure. 
In any case if you update it then it should just have the changes that you made.